pyramid_oereb/contrib/stats/scripts/create_stats_tables.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from c2cwsgiutils.sqlalchemylogger.handlers import SQLAlchemyHandler
+import configparser
+import ast
+from mako.template import Template
+import os
+import optparse
+
+SCRIPT_FOLDER = os.path.dirname(os.path.abspath(__file__))
+
+
+def create_stats_tables():
+    parser = optparse.OptionParser(
+        usage='usage: %prog [options]',
+        description='creates database and views for sqlalchemy_logger'
```

Comment:
Sometimes we write "sqlalchemylogger", sometimes "sqlalchemy_logger". Should be consistent. I guess "sqlalchemy_logger" is nicer.

Reply:
the module is called sqlalchemylogger in c2cwsgiutils, I will change that accordingly.

Reply:
modified so that they all are named according to the module, i.e. `sqlalchemylogger`